gatsby-node.js

```diff
@@ -37,7 +51,23 @@ function fetchExistingObjects(index, reporter, cache) {
     );
 }
 
-exports.onPostBuild = async function ({ graphql, reporter, cache }, config) {
+/**
+ * @typedef PluginConfiguration
+ * @property {string} appId
+ * @property {string} apiKey
+ * @property {import('algoliasearch').AlgoliaSearchOptions} algoliasearchOptions
+ * @property {any[]} queries
```

Comment:
`object[]` I think

Reply:
bit more detailed in https://github.com/algolia/gatsby-plugin-algolia/pull/162/commits/e9cabfa65a675699b7863efdddabdae8e0ee6a5b